hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,402 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "dcd",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i",
+  ],
+  reset_primary: "rst_ni",
```

Comment:
I think you might need `other_reset_list` also

Reply:
Will fix.